visualization/app/codeCharta/ui/mapTreeView/mapTreeView.level.component.ts

```diff
@@ -90,15 +85,15 @@ export class MapTreeViewLevelController implements BuildingHoveredSubscriber, Bu
 		return markingColor ? markingColor : defaultColor
 	}
 
-	public isSearched(): boolean {
-		if (this.node != null && this.storeService.getState().dynamicSettings.searchedNodePaths) {
+	public isSearched() {
+		if (this.node && this.storeService.getState().dynamicSettings.searchedNodePaths) {
```

Comment:
```suggestion
		if (this.storeService.getState().dynamicSettings.searchedNodePaths) {
```

Just tested this and I couldn't encounter a case where node is not set. Let's remove this safeguard

Reply:
I would rather keep this for now and maybe change it in another PR.

Reply:
I'll leave the conversations regarding removing safeguards, which need a test adaptation open. It's easier for me to create a follow up PR based on the open conversations then.